### ISSUE TITLE
Focal length estimate stays at -1.00 for the first few hundred frames

### ISSUE THREAD

**Issue by hisprofile** [thegoodhen/AprilTools#21]
When I estimate the focal length with AprilTools, the focal length estimate stays at -1.00. This doesn't change for a long time.

**Comment by pitibiribas** [comment]
hello, i was having the same issue. from what i understand, apriltools uses the tag's edges to calculate the vanishing point, so it usually works better when the tag is seen from perspective, like this
![video0112](https://user-images.githubusercontent.com/60185731/72903787-a0820980-3d0c-11ea-8b7c-1174000af843.png)

instead of a flat, frontal angle like that
![video0038](https://user-images.githubusercontent.com/60185731/72903784-9fe97300-3d0c-11ea-9fd6-40047da9ce5e.png)

anyway, hope i've helped you :^)

**Comment by thegoodhen** [comment]
Yup, exactly right. I really need to release a new version. Stuff keeps getting in the way. ;-; I have my finals in 2 weeks and ughhhh...
